lib/containers/artist.js

```diff
@@ -54,6 +62,12 @@ export default Relay.createContainer(Artist, {
   fragments: {
     artist: () => Relay.QL`
       fragment on Artist {
+        partner_shows {
+          _id
+        }
```

Comment:
Yeah I checked the counts endpoint and it didn't have shows :/

Sent from my iPhone

> On May 23, 2016, at 13:42, Eloy Durán <notifications@github.com> wrote:
> 
> In lib/containers/artist.js:
> 
> > @@ -54,6 +62,12 @@ export default Relay.createContainer(Artist, {
> >    fragments: {
> >      artist: () => Relay.QL`
> >        fragment on Artist {
> > +        partner_shows {
> > +          _id
> > +        }
> I worry that this might lead to much more data being fetched, which is why I suggested to use those query fragments that we already use: #76 (comment)
> 
> However, the best solution imo would be to make metaphysics be able to return a shows count, like you did for the artworks. Have you checked if it doesn’t provide that yet?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>